pkg/egressgateway/manager.go

```diff
@@ -185,7 +236,23 @@ func NewEgressGatewayManager(p Params) *Manager {
 		},
 	})
 
-	return manager
+	return manager, nil
+}
+
+func (manager *Manager) setEventBitmap(events ...eventType) {
+	for _, e := range events {
+		manager.eventsBitmap |= e
+	}
+}
+
+func (manager *Manager) eventBitmapIsSet(events ...eventType) bool {
+	for _, e := range events {
+		if manager.eventsBitmap&e != 0 {
+			return true
+		}
+	}
```

Comment:
You're telling me there is no `return (manager.eventsBitmap & mask) != 0` ? :smile:

Reply:
eh, you have to build it first that `mask` :see_no_evil: (now, we could discuss if `eventBitmapIsSet` _really_ needs to be variadic, but I'd say it makes the invocation a tiny bit more readable when you have multiple events)